docs/reference/ebpf-traffic-collection.md

```diff
@@ -0,0 +1,112 @@
+# eBPF Traffic Collection
+
+## What is eBPF?
+
+eBPF (extended Berkeley Packet Filter) is a  technology that allows running sandboxed programs in the Linux kernel without changing kernel source code or loading kernel modules. Originally designed for packet filtering, eBPF has evolved into a general-purpose execution engine that enables safe and efficient kernel programming. Many Kubernetes network technologies such as [Envoy](https://istio.io/latest/docs/overview/dataplane-modes/) and [Cilium](https://cilium.io/) are moving to eBPF to reduce resource utilization.
+
+## How eBPF Works for Network Traffic Collection
+
+### Kernel-Level Observability
+
+eBPF programs run directly in kernel space, providing unprecedented visibility into network traffic with minimal overhead. Unlike traditional packet capture methods that require copying packets from kernel to user space, eBPF can process network data directly where it originates.
+
+Key advantages include:
+- **Zero-copy operations**: Data can be analyzed without expensive memory copies
+- **Minimal performance impact**: Programs run in kernel space with native performance
+- **Real-time processing**: Traffic analysis happens as packets flow through the network stack
+- **Comprehensive visibility**: Access to both network packets and kernel metadata
+
+### Traffic Capture Architecture
```

Comment:
Probably should include a section on user-space uprobes/uretprobes making TLS visibility possible